vassal-app/src/main/java/VASSAL/tools/SequenceEncoder.java

```diff
@@ -178,7 +178,7 @@ public String nextToken() {
         int end = i;
         while (begin < end) {
           if (val.charAt(end - 1) == '\\') {
-            buffer.append(val.substring(begin, end - 1));
+            buffer.append(val, begin, end - 1);
```

Comment:
I know why it was like this: Originally this used a `StringBuffer`, and that overload of `append` didn't exist in `StringBuffer` until Java 5.